dataLayer Builder w-Standard eCommerce.tpl

```diff
@@ -0,0 +1,295 @@
+                  },
+                  {
+                    "type": 1,
+                    "string": "write"
+                  },
+                  {
+                    "type": 1,
+                    "string": "execute"
+                  }
+                ],
+                "mapValue": [
+                  {
+                    "type": 1,
+                    "string": "createQueue"
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
+                  }
+                ]
+              },
+              {
+                "type": 3,
+                "mapKey": [
+                  {
+                    "type": 1,
+                    "string": "key"
+                  },
+                  {
+                    "type": 1,
+                    "string": "read"
+                  },
+                  {
+                    "type": 1,
+                    "string": "write"
+                  },
+                  {
+                    "type": 1,
+                    "string": "execute"
+                  }
+                ],
+                "mapValue": [
+                  {
+                    "type": 1,
+                    "string": "dataLayer"
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
+                  },
+                  {
+                    "type": 8,
+                    "boolean": false
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    },
+    "clientAnnotations": {
+      "isEditedByUser": true
+    },
+    "isRequired": true
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+const makeTableMap = require('makeTableMap');
+const query = require('queryPermission');
+const createQueue = require('createQueue');
+const customParameters = data.customParameters ? makeTableMap(data.customParameters, 'name', 'value') : {};
+const standardParameters = data.standardParameters ? makeTableMap(data.standardParameters, 'name', 'value') : {};
+var merge = function() {
+    var obj = {},
+        i = 0,
+        il = arguments.length,
+        key;
+    for (; i < il; i++) {
+        for (key in arguments[i]) {
+            if (arguments[i].hasOwnProperty(key)) {
+                obj[key] = arguments[i][key];
+            }
+        }
+    }
+    return obj;
+};
+var dataLayer = merge(customParameters, standardParameters);
+if (query('access_globals', 'readwrite', 'dataLayer')) {
+  const dataLayerPush = createQueue('dataLayer');
+  dataLayerPush(dataLayer);
+  var log = require('logToConsole');
```

Comment:
This is an API declaration, so you should do it at the top of the template code just like all the API declarations. Also, use `const log = require('logToConsole');`.

Reply:
Have done as requested, the code is much cleaner now, thanks!